src/ingameWindows/iwBuilding.cpp

```diff
@@ -100,6 +103,32 @@ iwBuilding::iwBuilding(GameWorldView& gwv, GameCommandFactory& gcFactory, nobUsu
 
     AddText(10, DrawPoint(113, 50), _("(House unoccupied)"), COLOR_RED, FontStyle::CENTER, NormalFont);
 
+    if(gwv.GetWorld().GetGGS().isEnabled(AddonId::SHOWRESOURCES))
+    {
+		if(BuildingProperties::IsMine(building->GetBuildingType()))
```

Comment:
I dont see the reason why not share the code for mine and well? i would prefer the de-doubled code, instead the "optimized" call stack for wells.

Reply:
yep thats right, thats duplicated. why not simply add a `case BLD_WELL` ?